packages/containers/src/Form/Form.test.js

```diff
@@ -34,6 +35,79 @@ describe('Container(Form)', () => {
 		expect(wrapper.getElement()).toMatchSnapshot();
 	});
 
+	it('should render with prop uiform = false : Form', () => {
+		const wrapper = shallow(
+			<Container
+				formId="test-form"
+				jsonSchema={{ schema: true }}
+				uiSchema={{ uiSchema: true }}
+				actions={[]}
+				formProps={{ other: true }} // extra props
+				uiform={false}
+				language={key => key}
+			/>,
+		);
+		expect(wrapper.dive().getElement().type.name).toEqual(Container.name);
+	});
+
+	it('should render with prop uiform = true : UIForm', () => {
+		const wrapper = shallow(
+			<Container
+				formId="test-form"
+				jsonSchema={{ schema: true }}
+				uiSchema={{ uiSchema: true }}
+				actions={[]}
+				formProps={{ other: true }} // extra props
+				uiform
+			/>,
+		);
+		expect(
+			wrapper
+				.dive()
+				.dive()
+				.getElement().type.name,
+		).toEqual(UIForm.name);
+	});
+
+	it('should render UIForm with not language prop set', () => {
+		const wrapper = shallow(
+			<Container
+				formId="test-form"
+				jsonSchema={{ schema: true }}
+				uiSchema={{ uiSchema: true }}
+				actions={[]}
+				formProps={{ other: true }} // extra props
+				uiform
+				language={{ OBJECT_REQUIRED: 'Field translated' }}
+			/>,
+		);
+		expect(
+			wrapper
+				.dive()
+				.dive()
```

Comment:
Instead of `.dive()` that is not very clear, can you replace it by `.find(UIForm)` ?